### ISSUE TITLE
Edge Alpha

### ISSUE THREAD

**Issue by pauldix** [influxdata/influxdb#24650]
The Alpha will be the first public facing release of InfluxDB 3.0 Edge. For that we'll want to have a fully functioning server that has had some level of internal performance testing, public facing documentation, and public facing builds. It should include the ability to write data via the v1, v2, or v3 Line Protocol APIs, query via v1 InfluxQL API, v3 query_sql, and query_influxql, and FlightSQL and InfluxQL via Flight. It should persist all data to object storage (or a local disk). A stretch goal will be to include an embedded VM for processing data.
```[tasklist]
### Tasks
- [ ] https://github.com/influxdata/influxdb/issues/24740
- [ ] https://github.com/influxdata/influxdb/issues/24773
- [ ] https://github.com/influxdata/influxdb/issues/24780
- [ ] https://github.com/influxdata/influxdb/issues/24781
- [ ] https://github.com/influxdata/influxdb/issues/24815
- [ ] Getting started documentation
- [ ] Update README
- [ ] Homebrew builds
- [ ] Update CONTRIBUTING
- [ ] Basic Performance Testing
- [ ] End-to-end API tests
- [ ] V3 Write API & Data Model (alpha release not gated on this)
- [ ] Embedded VM (alpha release not gated on this)
```

**Comment by hiltontj** [comment]
@pauldix wondering if we should close this one out now... I would say this was somewhat superseded by the [launch tracker issue](https://github.com/influxdata/influxdb_pro/issues/334).

I will close but re-open if that's not right

**Comment by rmja** [comment]
@hiltontj that is not really helpful as the pro repo is not public...

**Comment by pauldix** [comment]
Yeah, this is irrelevant now that the alpha is out.
